client/packages/common/src/ui/components/inputs/Filters/DateFilter.tsx

```diff
@@ -19,14 +19,15 @@ export type RangeOption = 'from' | 'to';
 export const DateFilter: FC<{ filterDefinition: DateFilterDefinition }> = ({
   filterDefinition,
 }) => {
+  const currentYear = new Date().getFullYear()
   const {
     type,
     urlParameter,
     name,
     range,
     displayAs = type,
-    maxDate = '9999-12-31',
-    minDate = '0000-01-01',
+    maxDate = `${currentYear + 10}-12-31`,
+    minDate = `${currentYear - 120}-01-01`,
```

Comment:
Really good find for speeding this up! 👏

I'm just wondering if we need these default values at all? The DatePicker accepts `undefined` for min and max just fine, and I can't see anything immediately wrong with just removing the defaults altogether. Can you?

Reply:
You're right I agree - just looking now  and see the date picker defaults to cover 1900 -> 2099 which seems like reasonable bounds and covers a very early date of birth edge case.

I'll remove the defaults